bigquery/samples/export_data_to_cloud_storage.py

```diff
@@ -11,16 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import argparse
 import uuid
 
-from bigquery.samples.utils import get_service
-from bigquery.samples.utils import poll_job
-from six.moves import input
+from utils import get_service, poll_job
 
 
 # [START export_table]
 def export_table(service, cloud_storage_path,
-                 projectId, datasetId, tableId,
+                 project_id, dataset_id, table_id,
```

Comment:
Just to give my thinking behind this. Camel case matches the field names in the json objects. So you could make these kwargs and then do something like:
```
tableRef = fetch_some_bigquery_object()['tableReference']
export_table(service, cloud_storage, **tableRef)
```
Probably not worth inconsistent/camel case naming, but something to consider.